.changeset/rich-dodos-smell.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We have codeowners set up for changesets if you manually tweak the filename to `cost-insights-X.md`